code/__DEFINES/preferences.dm

```diff
@@ -28,6 +28,7 @@
 #define CHAT_NO_TICKETLOGS 65536
 #define UI_DARKMODE 131072
 #define DISABLE_KARMA 262144
+#define CHAT_NO_MENTORTICKETLOGS 524288
```

Comment:
This is ***disgusting***

At this point, we're abusing flags.

Reply:
Aye it's gross. Can do it in another way but hey it works

Reply:
![image](https://user-images.githubusercontent.com/25063394/54077601-9ef55180-42b2-11e9-8857-a1569017119b.png)

Reply:
If they didn't want us to abuse it, why would they let us abuse it? 

😄